src/content/docs/fr/2/guide/upgrade-migrate/from-tauri-1.mdx

```diff
@@ -0,0 +1,1148 @@
+---
+title: Mise à Niveau Depuis Tauri 1.0
+
+i18nReady: true
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import CommandTabs from '@components/CommandTabs.astro';
+
+Ce guide vous explique comment mettre à niveau votre application basée sur Tauri 1.0 vers Tauri 2.0.
+
+## Migration Automatisée
+
+
+Tauri v2 contient une commande `migrate` qui automatise la plupart des processus et vous aide à finir votre migration:
+
+<CommandTabs
+	npm="npm install @tauri-apps/cli@latest
+    npm run tauri migrate"
+	yarn="yarn upgrade @tauri-apps/cli --latest
+    yarn tauri migrate"
+	pnpm="pnpm update @tauri-apps/cli --latest
+    pnpm tauri migrate"
+	cargo='cargo install tauri-cli version "^2.0.0-alpha"
+    cargo tauri migrate'
+/>
+
+{/* TODO: 2.0 */}
+
+Apprenez-en plus à-propos de la commande `migrate` dans l'[interface de référence des commandes](/2/reference/cli#migrate) 
+Learn more about the `migrate` command in the [Command Line Interface reference](/2/reference/cli#migrate)
+
+## Sommaire des Changements
+
+
+Voici ci-dessous un sommaire des changement entre Tauri 1.0 et Tauri 2.0:
+
+### Configuration Tauri
+
+- `tauri > allowlist` retiré.
+- `tauri > allowlist > protocol > assetScope` déplacé vers `tauri > security > assetProtocol > scope`.
+- `tauri > cli` déplacé vers `plugins > cli`.
+- `tauri > updater` déplacé vers `tauri > bundle > updater`.
+- `tauri > updater > dialog` retiré.
+- `tauri > updater > endpoints` déplacé vers `plugins > updater`.
+- `tauri > systemTray` renomé en `tauri > trayIcon`.
+
+[Référence de l'API de configuration de Tauri 2.0](/2/reference/config)
+
+### Nouvelles Fonctionnalités de Cargo
+
+- linux-protocol-body: Permet les protocols de requêtes "body parsing", autorisant l'IPC à l'utiliser. Nécéssite webkit2gtk 2.40.
+
+### Fonctionnalités de Cargo Retirées

```

Comment:
Not sure about this one tbh

```suggestion
### Fonctionnalités Cargo Retirées
```

Reply:
Both are working, I guess yours is more concise